src/components/chip-set/chip-set.scss

```diff
@@ -21,14 +21,6 @@ $mdc-text-field-fullwidth-bottom-line-color: $lime-text-field-bottom-line-color;
  * @prop --input-chip-set-selected-chip-color: Color of the highlight around selected chips in input chip-sets.
  */
 
-:host([disabled]) {
-    pointer-events: none;
-
-    .mdc-chip-set {
-        opacity: 0.5;
-    }
-}
```

Comment:
Oops… removing this means it's no longer possible to disable chip-sets which are not of type `input`.

I'll fix, but don't merge for now!

Reply:
⚡️ (see new lines 31-35 below)